docs/vo_service_discovery/vo_service_discovery.rst

```diff
@@ -0,0 +1,183 @@
+.. doctest-skip-all
+
+.. _astroquery.vo_service_discovery:
+
+*************************************************************************
+Virtual Observatory Service Discovery (`astroquery.vo_service_discovery`)
+*************************************************************************
+
+Getting Started
+===============
+
+Archives from around the world have registered their services with the NASA
+Astronomy Virtual Observatory (NAVO) directory. A web interface is available
+at http://vao.stsci.edu/keyword-search/. This module provides methods
+to query this directory by service type, keywords, wavelength band or coverage,
+and source or publisher.
+
+Here is an example showing show to find information about conesearch services
+provided by the NASA Extragalactic Database (NED).
+
+.. code-block:: python
+
+    >>> from astroquery.vo_service_discovery import Registry
+    >>> results = Registry.query(source='ned', service_type='cone')
+    >>> print(results)
+
+                                waveband                         ... service_type
+        -------------------------------------------------------- ... ------------
+                                                        infrared ...   conesearch
+        radio#millimeter#infrared#optical#uv#euv#x-ray#gamma-ray ...   conesearch
+
+The result of the query is an `astropy.table.Table` object. The methods of `Table`
+can be used to inspect the search result.
+
+.. code-block:: python
+    >>> print(results.colnames)
+
+        ['waveband', 'short_name', 'ivoid', 'res_description', 'access_url',
+         'reference_url', 'publisher', 'service_type']
+
+The value of `'access_url'` specifies the endpoint for the service.
+
+Documentation of the service may be found in the `'reference_url'` values.
+We could use capabilities in the standard library to open the first url
+in a web browser.
+
+.. code-block:: python
+
+    >>> import webbrowser
+    >>> webbrowser.open(results['reference_url'][0])
```

Comment:
In the course of testing, I modified this to:
```suggestion
    >>> webbrowser.open(results['reference_url'][1].decode())
```
because the item is a byte-string, which  causes an exception in `web browser.open`.